components/MaskedTransition/src/private/MDCMaskedPresentationController.h

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@protocol MDMTransitionContext;
+
+FOUNDATION_EXPORT UIView *CreateScrimView(id<MDMTransitionContext> context);
```

Comment:
To clarify: I'm willing to push for updating the spec to remove the scrim behavior on full-screen transitions. The purpose of a scrim is to indicate that underlying content can't be directly interacted with. For a full-screen transition, this purpose is nulled, so I can argue that the scrim's appearance is unhelpful.